usr/sbin/pkcscca/pkcscca.mk

```diff
@@ -1,13 +1,51 @@
 sbin_PROGRAMS += usr/sbin/pkcscca/pkcscca
 noinst_HEADERS += usr/sbin/pkcscca/pkcscca.h
+noinst_HEADERS += usr/lib/common/defs.h
+noinst_HEADERS += usr/lib/common/host_defs.h
+noinst_HEADERS += usr/include/local_types.h
+noinst_HEADERS += usr/lib/common/h_extern.h
+noinst_HEADERS += usr/lib/common/pkcs_utils.h
 
-usr_sbin_pkcscca_pkcscca_LDFLAGS = -lcrypto -ldl
+usr_sbin_pkcscca_pkcscca_LDFLAGS = -lcrypto -ldl -lrt
 
 usr_sbin_pkcscca_pkcscca_CFLAGS  =					\
 	-DSTDLL_NAME=\"pkcscca\"					\
-	-I${srcdir}/usr/include -I${srcdir}/usr/lib/common		\
+	-DTOK_NEW_DATA_STORE=0x0003000c					\
+	-DLINUX -DNOCDMF -DNODSA -DNODH -DNOECB				\
```

Comment:
I don't find any usages of `-DLINUX` and `-DNOECB`.  Do we really need them?  Also, seeing that we are using a configure script, shouldn't `-DLINUX` come via this script?

Reply:
I copied this from the CCA token make file. But you are right both are not used anywhere. So at least I will remove them here. We might want to clean up all of those at other places in a separate PR....